pyproject.toml

```diff
@@ -20,6 +20,7 @@ litellm = "^1.59.8"
 chevron = "^0.14.0"
 llamux = "^0.1.9"
 pillow = "^11.1.0"
+hvac = "^2.3.0"
```

Comment:
this should be an optional dependency I think. Not all users would want this to be installed by default